File: Estructuras/Arrays/Array.py

arr = [1, 2, 3, 4, 5]

for i in range(5): # Example of how we can see the address of an array's elements in python.
    print('Element {} is in address: {}'.format(arr[i], id(arr[i]))) # Huh, formato print similar a C.

File: Estructuras/Arrays/Array.c
#include <stdio.h>

int main() 
{
    int arr[5] = {1, 2, 3, 4, 5};
    int i; // Initialize to understand where i is located.

    printf("Counter i is in address: %d\n", (int) &i); 

    for (i = 0; i < 5; i++) // For loop to understand data size in RAM using the address for elements in an array.
    {
        printf("Element %d is in address: %d\n", arr[i], (int) &arr[i]); 
    }
}